### PR TITLE
Remove promptTemplateId field from reflect stage

### PR DESCRIPTION
**Pull request by Dschoordsch**
Remove the field from the meeting stage. The template is now not
connected to the stage anymore so update the template timestamp where it
is used when starting the meeting.

Resolves: #3949

### CODE REVIEW

**Comment by mattkrick** [comment]
Whoa, that was fast! :tada: 
When you're ready for a review can you select one from the right (maybe @nickoferrall ) and at-mention them here? 
We'll track the progress based on the issue the PR fixes, so feel free to drag #3949 to the "Reviewer Review" column when ready.


We should be able to get this one merged pretty quickly!

**Comment by Dschoordsch** [comment]
@nickoferrall could you please take a look?

**Review by nickoferrall — APPROVED** [review]
Fantastic work! This LGTM so I'll tag in Matt and move the issue to the Maintainer Review column in the Sprint Board

**Review by mattkrick — APPROVED** [review]
stellar work! both migrations are _really_ well written.
one tiny nit on running the DB hits in parallel. Merge when ready!

File: packages/server/database/migrations/20210817223727-removePromptTemplateId.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export const down = async function(r: R) {
+  try {
+    await r
+      .table('NewMeeting')
+      .filter({meetingType: 'retrospective'})
+      .update((meeting) => ({
+        phases: meeting('phases').map((phase) =>
+          phase('phaseType')
+            .eq('reflect')
+            .branch(phase.merge({promptTemplateId: meeting('templateId')}), phase)
```
[the remark this answers is not in the document]
Comment (Dschoordsch) [reply packages/server/database/migrations/20210817223727-removePromptTemplateId.ts:26]: Thanks. I'm certain I haven't used it before, I probably spent an hour just to get this down migration right. But I figured this is still worth learning.

**Pull request merged**